src/main/java/core/selenium/WebDriverConfig.java

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright (c) 2021 Fundacion Jala.
+ * This software is the confidential and proprietary information of Fundacion Jala
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ */
+
+package core.selenium;
+
+import core.utils.PropertiesReader;
+
+import java.util.Properties;
+
+public final class WebDriverConfig {
+
+    private String browser;
+    private int implicitWaitTime;
+    private int explicitWaitTime;
+    private int waitSleepTime;
+
+    private static WebDriverConfig instance;
+
+    /**
+     * Constructor of page WebDriverManager.
+     */
+    private WebDriverConfig() {
+        initialize();
+    }
+
+    /**
+     * Constructo of WebDriverConfig.
```

Comment:
I would be better instances or build or some thing like that.Use third person in verbs